MAINTAINERS.md

```diff
@@ -66,9 +66,9 @@ To cut a new release:
 
 ## Publishing a release on [nixpkgs][]
 
-lorri is available in the `nixos-unstable` and `nixos-19.09` release channels,
-which correspond to the `master` and `release-19.09` branches in the
-[nixpkgs][] repository, respectively.
+lorri is available in the `nixos-unstable` and the stable release channels,
```

Comment:
"the stable release channels" would include 18.09. lorri is not currently in there. Are you saying that we should backport lorri to 18.09?

Reply:
Ah, I didn’t want to say all stable channels of course. Rewording.